libs/sdk-backend-bear/src/backend/workspace/elements/elements.ts

```diff
@@ -46,6 +60,18 @@ class BearWorkspaceElementsQuery implements IElementQuery {
         return this;
     }
 
+    public withAttributeFilters(
+        attributeRef: ObjRef,
+        filters: IElementQueryAttributeFilter[],
+    ): IElementQuery {
+        if (!isUriRef(attributeRef)) {
+            throw new NotSupported("Identifier refs are not supported in elements attribute filter");
```

Comment:
Why are they not supported? You can convert idRefs to uriRefs if needed with `objRefToUri`.